### ISSUE TITLE
Chrome/Edge Bookmarks Bar != Firefox Bookmarks Toolbar

### ISSUE THREAD

**Issue by stutopp** [floccusaddon/floccus#1733]
### Which version of floccus are you using?

5.3.0

### How many bookmarks do you have, roughly?

125

### Are you using other means to sync bookmarks in parallel to floccus?

no

### Sync method

Google Drive

### Which browser are you using? In case you are using the phone App, specify the Android or iOS version and device please.

Chrome 129.0.6668.71, Edge 129.0.2792.65, Firefox 130.0.1

### Describe the Bug

In Chrome/Edge, the bookmarks toolbar is called "Bookmarks bar". In Firefox, the toolbar is called "Bookmarks Toolbar". Therefore, only Chrome/Edge or Firefox can sync from the bookmarks root, and the other must be nested in a single folder on the toolbar or in other bookmarks.

### Expected Behavior

I expect my bookmarks to sync from root to root on supported browsers, with bookmarks on the toolbar being placed on the toolbar and other bookmarks being placed in other bookmarks.

### To Reproduce

1. Create a floccus sync from root in Chrome/Edge or Firefox.
2. Sync to other browser.
3. Bookmarks will be placed in a floccus folder under other bookmarks.
4. OR, you can set the sync location to root and return a failure.

### Debug log provided

- [ ] I have provided a debug log file

**Comment by marcelklehr** [comment]
I cannot reproduce this. I am aware that `Chrome/Edge Bookmarks Bar != Firefox Bookmarks Toolbar` and have added a mapping in the code that should take care of this and does so in my tests. Perhaps you could post the error message here?

**Comment by jaimehrubiks** [comment]
I'm assuming we are doing something wrong, but definitely this happens to me as well. I'm new to floccus and just created a profile in chrome. Then, I have this same issue both with Firefox and Zen Browser:

<img width="807" alt="image" src="https://github.com/user-attachments/assets/219f5f3b-1717-415a-936c-62c0e008aa20">

My chrome bookmarks are under Bookmarks Menu/Floccus (profile name)/Bookmarks bar. Instead of under Bookmarks Toolbar.

Should I choose a different set of options when creating the profile? @marcelklehr

**Comment by marcelklehr** [comment]
> Should I choose a different set of options when creating the profile?

What's your current Local folder option in the various browsers?

**Comment by jaimehrubiks** [comment]
Thanks for pointing in the right direction, I found the issue. I used the export/import to file function, and I thought that the same settings would be in the new browser, I didn't even get a prompt to change the settings, so I assumed so. I changed the new browsers to "/" (root) and it started working fine.

**Comment by marcelklehr** [comment]
![image](https://github.com/user-attachments/assets/bca266ba-0f00-4529-ba7e-ff93daef1ad3)

**Comment by stutopp** [comment]
@marcelklehr I believe your updated mappings fixed it. Upon import, it was a "folder not found" error (or something similar). I was able to select the root "Untitled folder" in the local mapping and the sync worked as expected. 

Thanks for your prompt attention and update! Resolved on my end.

<img width="673" alt="Capture" src="https://github.com/user-attachments/assets/b94f2446-fc0a-4bb9-82e4-4b76ab216780">
